pkg/quota/gcp/limits.go

```diff
@@ -8,7 +8,7 @@ import (
 	serviceusage "google.golang.org/api/serviceusage/v1beta1"
 )
 
-// loadLimis loads the comumer quota metric for a given project and the list of services.
+// loadLimits loads the comumer quota metric for a given project and the list of services.
```

Comment:
Maybe also fix another typo?
```suggestion
// loadLimits loads the consumer quota metric for a given project and the list of services.
```